inc/instance-config.php

```diff
@@ -559,3 +560,32 @@
 $config['max_body'] = 80000;
 $config['additional_javascript_compile'] = true;
 $config['minify_js'] = true;
+
+/*
+* ====================
+*  Ban settings
+* ====================
+*/
+
+    // Require users to see the ban page at least once for a ban even if it has since expired.
+    $config['require_ban_view'] = true;
+
+    // Show the post the user was banned for on the "You are banned" page.
+    $config['ban_show_post'] = true;
+
+    // Optional HTML to append to "You are banned" pages. For example, you could include instructions and/or
+    // a link to an email address or IRC chat room to appeal the ban.
+    $config['ban_page_extra'] = '';
```

Comment:
Requesting the ban_page_extra information about the rules and appeals be added to this commit before merging.